response_operations_ui/views/messages.py

```diff
@@ -449,6 +451,76 @@ def close_conversation(thread_id):
                            business_id_filter=business_id_filter)
 
 
+@messages_bp.route('/miscellaneous', methods=['GET', 'POST'])
+@login_required
+def view_misc_inbox():  # noqa: C901
+    session['messages_survey_selection'] = "misc"
+    breadcrumbs = [{"text": "Miscellaneous" + " Messages"}]
+
+    page = request.args.get('page', default=1, type=int)
+    limit = request.args.get('limit', default=10, type=int)
+    conversation_tab = request.args.get('conversation_tab', default='open')
+    ru_ref_filter = request.args.get('ru_ref_filter', default='')
+    business_id_filter = request.args.get('business_id_filter', default='')
+    category = "MISC"
+
+    form = SecureMessageRuFilterForm()
+
+    try:
+        if form.validate_on_submit():
```

Comment:
Is it possible to refactor this method? 

It's pretty much repeated three times in the code and this function is almost an identical copy of the view_technical_inbox method but with a different category. 

Maybe we could craete one method which takes a category and that's called by two different view functions?